### ISSUE TITLE
Accordion: Low contrast hover

### ISSUE THREAD

**Issue by EldRoos** [seb-oss/green#941]
### Bug already reported ?

- [X] I confirm that I have checked if the bug already has been reported

### Component name

Accordion

### Description

The hover-state of accordion does not change much - should have higher contrast. Would help a11y.
The code follows Figma - but issue have been created there too to change this.

**Comment by EldRoos** [comment]
Looks like it has been fixed since last checked (about one month ago). Contrast now have a more visible in chlorophyll/react/angular storybook on hover.
